webapp/src/ts/app.component.ts

```diff
@@ -643,4 +651,20 @@ export class AppComponent implements OnInit {
     // avoid Failed to fetch errors being logged when the browser window is reloaded
     this.changesService.killWatchers();
   }
+
+  private async initAnalyticsModules() {
+    try {
+      const canAccess= await this.headerTabsService.canAccessTab('analytics');
+
+      if (!canAccess) {
+        return;
+      }
+
+      const modules = await this.analyticsModulesService.get();
```

Comment:
We have to load the modules anyway. The `can_view_**_tab` doesn't stop the user from seeing the page, it just hides the link from header tabs, but the link will appear in the hamburger menu. 
Right now, if I remove `can_view_analytics_tab` from a role, but keep the `can_view_analytics`, then clicking on the link in the hamburger menu will do nothing. 
If I manually redirect to `/#/analytics` and refresh the page, it will load an empty page.

Reply:
oh I see, okay then we'll load the modules always 👍

Reply:
Done, loading modules always